python/jiminy_py/src/jiminy_py/viewer.py

```diff
@@ -507,103 +700,168 @@ def _updateGeometryPlacements(self, visual=False):
         else:
             geom_model = self._rb.collision_model
             geom_data = self._rb.collision_data
-
         pin.updateGeometryPlacements(self.pinocchio_model,
                                      self.pinocchio_data,
                                      geom_model, geom_data)
 
-    def setCameraTransform(self, translation=np.zeros(3), rotation=np.zeros(3), relative=False):
+    def set_camera_transform(self, translation=None, rotation=None, relative=None):
         # translation : [Px, Py, Pz], rotation : [Roll, Pitch, Yaw]
+        # If no translation or rotation are set, initialize camera towards origin of the plane
+
+        # Handling of default transformation
+        if translation is None and rotation is None:
```

Comment:
It would be better to set independently the translation and rotation, and use different default if relative is not 'camera'.